regular_expressions_boundaries_anchors/regular_expressions_boundaries_anchors.md

```diff
@@ -0,0 +1,419 @@
+@end
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Explain what a regular expression flag does
+- Use anchors and boundaries in regular expressions
+- Use boundaries in regular expressions
+
+@end
+good_first_module: false
+coding_required: true
+coding_level: intermediate
+sequence_name: regex
+previous_sequential_module: regular_expressions_groups
+@sets_you_up_for
+- regular_expressions_lookaheads
+@end 
+@depends_on_knowledge_available_in
+- demystifying_regular_expressions
+- regular_expressions_basics
+- regular_expressions_groups
+@end
+@version_history 
+No previous versions.
+@end
+
+import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros.md
+-->
+
+# Regular Expressions: Flags, Anchors, and Boundaries
+
+@overview
+
+## Regular Expression Refresher
+
+A **regular expression** (also known as "regex") is a specific way to **express** a **rule** for a pattern.  Regular expressions are useful to biomedical researchers because they can allow you to find, replace, or extract text that matches patterns you define.  
+
+A good place to practice regular expressions is an online regex testing website.  We like **Regex 101**, at https://www.regex101.com.
+
+Regex consists of **tokens** which represent characters that appear in a particular location, and **quantifiers**, which give the number of times a given token repeats.
+
+- **Tokens** include `\d`, `[A-Za-z]`, `A`, and `.`
+
+- **Quantifiers** include `*`, `{2,5}`, and `+`
+
+Regular expressions can make use of **groups**, both **capturing** and **non-capturing**.
+
+If this refresher doesn't seem familiar, you might want to consider reviewing [Demystifying Regular Expressions](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/demystifying_regular_expressions/demystifying_regular_expressions.md#1), [Regular Expressions Basics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/regular_expressions_basics/regular_expressions_basics.md#1), and [Regular Expressions: Groups](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/regular_expressions_groups/regular_expressions_groups.md#1).
+
+## Scope of This Module
+
+We consider this module to be intermediate, because it assumes basic knowledge covers more advanced regular expressions topics, without delving into the advanced topic of using regular expressions within the context of code.  
+
+Specifically, we'll address these topics:
+
+* **Anchors**, which help you define patterns that belong in the beginning and end of a string as a whole
+* **Boundaries**, which help you define patterns that belong in certain places that appear within a string
+* **Exclusions**, which allow you to look for patterns that exclude certain characters
```

Comment:
This doesn't match the table of contents. I'd be okay with taking this page out entirely, but I think if it is there it should match the order and relative importance of each topic indicated by the table of contents.